packages/angular_devkit/build_angular/plugins/webpack/analytics.ts

```diff
@@ -156,7 +159,16 @@ export class NgBuildAnalyticsPlugin {
       // This does not include View Engine AOT compilation, we use the ngfactory for it.
       this._stats.numberOfComponents += countOccurrences(module._source.source(), ' Component({');
       // For Ivy we just count ngComponentDef.
-      this._stats.numberOfComponents += countOccurrences(module._source.source(), 'ngComponentDef', true);
+      const numIvyComponents = countOccurrences(module._source.source(), 'ngComponentDef', true);
+      this._stats.numberOfComponents += numIvyComponents;
+
+      // Check whether this is an Ivy app so that it can reported as part of analytics.
+      if (!this._stats.isIvy) {
+        const hasIvyNgModule = module._source.source().indexOf('ngModuleDef') >= 0;
```

Comment:
Nit: you can use `.includes` instead of `indexOf`

Reply:
Done.